packages/labs/react/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 1.2.0
+
+### Minor Changes
+
+- [#3885](https://github.com/lit/lit/pull/3885) [`7932f7dd`](https://github.com/lit/lit/commit/7932f7ddc21308dc0bf7b1bbd0dde781a6c8dece) - `@lit-labs/ssr-react` and `@lit-labs/react` now coordinate prop handling during server rendering and hydration.
+
+  - [Breaking] `@lit-labs/ssr-react` will call `setAttribute()` with all props provided during server rendering unless they are provided in `_$litProps# Change Log object made by `@lit-labs/react`. Previously, it would call `setProperty()`for props that were found in the element's prototype, and`setAttribute()`for those that weren't. If you wish to server render custom elements that take properties that can not be set as attributes (i.e. not serializeable or have different name as attribute/property), you must use`@lit-labs/react` to create a React wrapper component.
```

Comment:
Am I reading this correctly? Why is the text `Change Log object made by @lit-labs/react` present in here?

Reply:
Weird.. that's definitely not right.